src/utils/bounding-volume-from-tile.ts

```diff
@@ -0,0 +1,40 @@
+import type { ObbData } from './types';
```

Comment:
I suggest joining this file and `get-volume-type` in `bounding-volume.ts`

Reply:
Done